ExampleMod/Content/Projectiles/ExampleSpearProjectile.cs

```diff
@@ -0,0 +1,94 @@
+﻿using ExampleMod.Content.Dusts;
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.ModLoader;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Content.Projectiles
+{
+	public class ExampleSpearProjectile : ModProjectile
+	{
+		public override void SetStaticDefaults() {
+			DisplayName.SetDefault("Spear");
+		}
+
+		public override void SetDefaults() {
+			projectile.width = 18; // The width of the projectile's hitbox in pixels.
+			projectile.height = 18; // The height of the projectile's hitbox in pixels.
+			projectile.aiStyle = 19; // Selects which vanilla code to use for the AI method. Modders can use vanilla aiStyle to mimic AI code already in the game.
+			projectile.penetrate = -1; // The Peneration of the projectile. -1 is infinite peneration.
+			projectile.scale = 1.3f; // The size multiplier of the projectile's sprite while the item is being used.
+			projectile.alpha = 0; // How transparent to draw this projectile. 0 to 255. 255 is completely transparent.
+
+			projectile.hide = true; 
```

Comment:
This could do with a comment (since I don't know what hide does off the top of my head)